plugin/formatting.py

```diff
@@ -181,14 +186,15 @@ class LspFormatDocumentRangeCommand(LspTextCommand):
     def is_enabled(self, event: dict | None = None, point: int | None = None) -> bool:
         if not super().is_enabled(event, point):
             return False
-        if has_single_nonempty_selection(self.view):
-            return True
-        if self.view.has_non_empty_selection_region() and \
-                bool(self.best_session('documentRangeFormattingProvider.rangesSupport')):
-            return True
-        return False
+        return bool(
+            has_single_nonempty_selection(self.view)
+            or
+            (self.view.has_non_empty_selection_region() and self.best_session('documentRangeFormattingProvider.rangesSupport'))  # noqa: E501
+        )
```

Comment:
To be honest I'm not a big fan of these stylistic changes to return `bool(...)` with these long expressions/function names. I think the `if ... return` is easier to read here.
Also the `bool` is technically only needed around the `self.best_session` because the two other functions already return a bool. (Alternatively it looks like the bool could be removed alltogether in the `if` condition in the original code.)
And the parenthesis after `or` are not necessary because `and` has higher precedence in Python (and in every other programming language that I know).